proto/generated-go/v1/org_policy_service.pb.go

```diff
@@ -250,6 +250,57 @@ func (MaskingExceptionPolicy_MaskingException_Action) EnumDescriptor() ([]byte,
 	return file_v1_org_policy_service_proto_rawDescGZIP(), []int{13, 0, 0}
 }
 
+type DataSourceQueryPolicy_Restricton int32
```

Comment:
Missing `buf generate` in https://github.com/bytebase/bytebase/commit/88429fcf164124ee90ec0cde31480ba956d5f2e1?